crates/turbopack/src/evaluate_context.rs

```diff
@@ -49,31 +50,36 @@ pub async fn node_evaluate_asset_context(
         } else {
             "development".to_string()
         };
+    let resolve_options_context = ResolveOptionsContext {
```

Comment:
Can we add some newlines and maybe a comment to the rules block below, took me a second to figure out why this is split into 2 declarations